packages/logger/src/index.ts

```diff
@@ -9,59 +9,59 @@ export enum LogLevel {
 }
 
 /**
- * Interface for objects where objects in this package's logs can be sent (can be used as `logger` option).
+ * Logging interface exposing different methods to log at different {@link LogLevel}s.
```

Comment:
By using `{@link}` inline tags, the generated reference documentation figures out what to link to 😍 
<img width="588" alt="Screenshot 2024-10-03 at 3 49 25 PM" src="https://github.com/user-attachments/assets/e6a45c7b-dce0-4532-938b-6b53134cc11e">